modules/nextflow/bin/main/nextflow/NextflowMeta.groovy

```diff
@@ -0,0 +1,203 @@
+package nextflow
+
+import java.text.SimpleDateFormat
+import java.util.regex.Pattern
+
+import groovy.transform.CompileStatic
+import groovy.transform.EqualsAndHashCode
+import groovy.transform.ToString
+import groovy.util.logging.Slf4j
+import nextflow.exception.AbortOperationException
+import nextflow.util.VersionNumber
+import static nextflow.extension.Bolts.DATETIME_FORMAT
+
+/**
+ * Models nextflow script properties and metadata
+ * 
+ * @author Paolo Di Tommaso <paolo.ditommaso@gmail.com>
+ */
+@Slf4j
+@CompileStatic
+@Singleton(strict = false)
+@ToString(includeNames = true)
+@EqualsAndHashCode
+class NextflowMeta {
```

Comment:
umm, I think something went wrong. There are new source files in this PR

Reply:
@llewellyn-sl it looks like you edited these files in `modules/nextflow/bin/main/nextflow` rather than `modules/nextflow/src/main/nextflow`. I think these files are duplicated during the build process so you may have gotten them mixed up. I would try copying these files back to their original location under `src` and see if the Git changes look right.

Reply:
Thanks. I have reverted the commit and will try another this week.